### ISSUE TITLE
Fix Airways

### ISSUE THREAD

**Issue by erikquinn** [openscope/openscope#627]
One of the FMS overhauls broke airways. Unlike the complex SIDs/STARs, airways only need an array of fix names, and aircraft can enter and exit the airway at any point, whether forward or backward through the array.

We need to somehow implement airways, as they are critical to improving the infrastructure of airspaces used in the simulator.

**Comment by n8rzz** [comment]
Could you include a brief example of how an airway would be defined, and subsequently how it would be used within a command?

**Comment by erikquinn** [comment]
It's actually already in the airport specific, looks like this:
```
airways: {
    V1: ["FIXXA", "FIXXB", "FIXXC", "FIXXD", FIXXE"],
    J511: ["FIXXL", "FIXXH", "FIXXB", "FIXXY", "FIXXW"]
}
`
And then it is used in a routestring. May enter and exit at any point, and the fixes between the entry/exit (in the correct direction) are added to the LegModel.

This would include spawn pattern route strings in airport files, route and reroute commands, etc. Anywhere a route can be specified in dot form, airways and procedures can be used. As a side note, it is very common to hop between procedures, eg `entry.procedureA.link1.procedureB.link2.procedureC.exit..entry.procedureD.link3.procedureE.exit`

There are no alt/spd restrictions, rwy segments or other such things. It would literally be a simple array of fixes encapsulated in a class with some simple helper methods. Since it's way simpler than the other procedures though, it doesn't seem like it would make sense being an implemention of a StandardRouteModel.

**Comment by n8rzz** [comment]
Awesome. That's what I thought, but figured I'd ask just in case. 

Hopefully, I can get some time in the next few days to add some tests and implement this. There are some sneaky things in this that will make things challenging.

**Comment by erikquinn** [comment]
Resolved by #846. Closing.
